lua/nordic/colors/nordic.lua

```diff
@@ -25,10 +25,10 @@ local palette = {
     gray5 = '#60728A',
 
     -- Dim white.
-    -- `white0` is used as the default fg, and has a blue tint.
-    -- Reduce that amount of tint.
-    white0_alt = '#C0C8D8',
-    white0 = '#BBC3D4',
+    -- default fg, has a blue tint.
+    white0_normal = '#BBC3D4',
+    -- less blue tint
+    white0_reduce_blue = '#C0C8D8',
```

Comment:
Rather not change these names, they are perfectly fine.

Reply:
If you are going to be able to reload the config, both values need to be separate.

You were leaving `white0` the same and adding `white_alt` (or something like that), however once you set white0 to alt you lose the original value of white and can't go back without closing and reopening neovim.

Reply:
Sure, but leave the names "white0" and "white0_alt".

Reply:
I can leave white0_alt but I would have to change white0.

I guess I could save a bool for if they are swapped right now but that seems like more of a mess.

Reply:
The actual value of `white0` is the same its just set in `extend_palette` instead.

I just changed the names to `white0_normal` & `white0_alt` but I think its more understandable like this.

I could ensure that `white0` exists even if you import from `colors.nordic` (no `extend_palette`) but that should never happen. Unless you use the values in the default config but then I think its better that you have to explicitly use one or the other.

Reply:
Oh I see now.  Then it is fine.